shell/models/workload.js

```diff
@@ -618,4 +619,26 @@ export default class Workload extends WorkloadService {
 
     return matching(allInNamespace, selector);
   }
+
+  get duration() {
```

Comment:
I believe this should be applied to the model `batch.job` rather than `workload` because `workload` is the parent class to all "workload" types (jobs, deployments, daemonsets, cronjobs,etc) and it looks to me this is only applied to `batch.job`

Reply:
Agree with you, moved to batch.job, thanks!